### PR TITLE
Fix issue 14431: Document alternative to CSSRule.type

### LINKED ISSUE #14431: Content bug: Recommend using  .constructor.name instead of deprecated CSSRule.type
## What page(s) did you find the problem on?

https://developer.mozilla.org/en-US/docs/Web/API/CSSRule/type

## Specific page section or heading?

CSSRule.type

## What is the problem?

If you find yourself needing _distinguish_ between different types of **CSSRule** and `.type` is deprecated, `.constructor.name` is a good alternative. Taken from [[all] Can we start ignoring CSSRule.type? ](https://github.com/w3c/csswg-drafts/issues/4142)

## What did you expect to see?

An additional note (**if** is the correct place to do so) about what can be used instead to identify different rules.

### PR DESCRIPTION
**Pull request by wbamberg**
Fixes https://github.com/mdn/content/issues/14431.

Thanks @iamzapata for filing the issue and providing some sample code for me to steal.

### CODE REVIEW

File: files/en-us/web/api/cssrule/type/index.md [hunk 1]
```diff
@@ -16,6 +16,17 @@ The read-only **`type`** property of the
 {{domxref("CSSRule")}} interface is a deprecated property that returns an integer
 indicating which type of rule the {{domxref("CSSRule")}} represents.
 
+If you need to distinguish different types of CSS rule, a good alternative is to use `constructor.name`:
```
Comment (teoli2003) [inline files/en-us/web/api/cssrule/type/index.md:19]: Do we have `constructor.name` documented somewhere?

I would like to have a link here: I was curious to see its browser compatibility but was unable to locate it.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/cssrule/type/index.md (hunk 1 above)
Comment (wbamberg) [reply files/en-us/web/api/cssrule/type/index.md:19]: I wondered about including a link, I wasn't sure not sure how helpful it is but am happy to.

I think `constructor` would be https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/constructor and `constructor.name` would be https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/name , no?

Comment (teoli2003) [reply files/en-us/web/api/cssrule/type/index.md:19]: I would never have found it!

What about:
```suggestion
If you need to distinguish different types of CSS rule, a good alternative is to use [`constructor.name`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/name):
```

**Review by teoli2003 — APPROVED** [review]
Thanks, @wbamberg

**Pull request merged**